docs/en_us/internal/testing.rst

```diff
@@ -281,7 +281,7 @@ console::
     paver test_js_run -s common
     paver test_js_run -s common-requirejs
 
-To run JavaScript tests in a browser:
+To run JavaScript tests in a browser::
```

Comment:
Hi @vkaracic, our preferred style is to use complete sentences and end them with periods. I suggest, 
To run JavaScript tests in a browser, run these commands.

::

  paver test_js...